workflows/snvphylnfc.nf

```diff
@@ -87,23 +87,32 @@ workflow SNVPHYL {
     // NB: `input` corresponds to `params.input` and associated sample sheet schema
     input = Channel.fromSamplesheet("input")
         // Map the inputs so that they conform to the nf-core-expected "reads" format.
-        // Either [meta, [fastq_1]] or [meta, [fastq_1, fastq_2]] if fastq_2 exists
-        .map { meta, fastq_1, fastq_2 ->
-               fastq_2 ? tuple(meta, [ file(fastq_1), file(fastq_2) ]) :
-               tuple(meta, [ file(fastq_1) ])}
+        // Either [meta, [fastq_1], assembly]
+        // or [meta, [fastq_1, fastq_2], assembly] if fastq_2 exists
+        .map { meta, fastq_1, fastq_2, assembly ->
+               fastq_2 ? tuple(meta, [ file(fastq_1), file(fastq_2) ], assembly) :
```

Comment:
Should `assembly` be wrapped by `file()` like the reads here?

Reply:
Added in 1da07b49d29bf659c2091e6ffe2dce4028688962